tests/logs/RegressionTests_hercules.log

```diff
@@ -7718,3 +7718,61 @@ Test 002 hafs_regional_storm_following_1nest_atm_ocn_wav_inline_intel PASS
 REGRESSION TEST WAS SUCCESSFUL
 Fri Feb  2 09:50:52 AM CST 2024
 Elapsed time: 00h:40m:07s. Have a nice day!
+Mon Feb  5 10:55:10 AM CST 2024
```

Comment:
I thought we were committing the logs for the two new tests appended to the existing log for this PR?

Reply:
Two cases are appended to the existing full test results if you see the log files.

Reply:
Yes, the full current results should still be there. They're just collapsed.